pom.xml

```diff
@@ -27,7 +27,7 @@
 				<plugin>
 					<groupId>org.jboss.tools.tycho-plugins</groupId>
 					<artifactId>target-platform-utils</artifactId>
-					<version>0.21.0</version>
+					<version>0.22.1-SNAPSHOT</version>
```

Comment:
I just wanted to know if/why it was done together with mylyn change. whether its done via parent pom or not is not really the point; it just looks weird/inconsistent to have some versions defined in properties and others not...makes it look as if they are "special". +1 for the change now that p2diff verifies nothing done but seems you already merged.